babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -1535,7 +1535,7 @@ private void checkParseInfixCast(String sqlType) {
   }
 
   @Test public void testPrimaryIndexMultipleOverwritten() {
-    final String sql = "create table foo primary index bar (lname) primary index baz (fname)";
+    final String sql = "create table foo primary index bar (lname), primary index baz (fname)";
```

Comment:
This is technically an edge case (since primary indices are only meant to be declared once), but if they were to be declared multiple times, they would need to be comma separated to conform to the syntax diagram, so I updated this test as well.